conjure-api/src/main/conjure/conjure-api.yml

```diff
@@ -173,13 +173,24 @@ types:
           auth: optional<AuthType>
           args: list<ArgumentDefinition>
           returns: optional<Type>
+          errors: list<EndpointError>
           docs: optional<Documentation>
           deprecated: optional<Documentation>
           markers: list<Type>
           tags: set<string>
       EndpointName:
         alias: string
         docs: Should be in lowerCamelCase.
+      EndpointError:
```

Comment:
Can we rename this to `ErrorTypeName`? It's basically equivalent to `TypeName`, but for errors

Reply:
sorry: this type makes sense, but should encapsulate a pair of ErrorTypeName (package/namespace/name) + docs

Reply:
Updated!